searx/static/themes/oscar/less/logicodev/variables.less

```diff
@@ -1,4 +1,5 @@
 @black: #29314D;
+@dim-blue: #2547bc;
```

Comment:
That's a small thing, but can you change `2547bc` by `2547BC` ?
It's just to have all colors written in the same way, thanks !

Reply:
Yes sure, no problem